app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -843,7 +843,7 @@ void handleScrollEvent(final int aHandle, final int aDevice, final float aX, fin
             }
             if (widget != null) {
                 float scrollDirection = mSettings.getScrollDirection() == 0 ? 1.0f : -1.0f;
-                MotionEventGenerator.dispatchScroll(widget, aDevice, aX * scrollDirection, aY * scrollDirection);
+                MotionEventGenerator.dispatchScroll(widget, aDevice, -aX * scrollDirection, aY * scrollDirection);
```

Comment:
I think we only want to invert it for the Quest joystick, I think it is correct for touch pad based controllers.

Reply:
I've verified that it should only be inverted for joysticks. Not sure where that fix belongs. Our horizontal scrolling on Go is the same as the Oculus browser.

Reply:
Done. I also verified that WebVR axes matches the Oculus behavior